src/Workspaces/MSBuild/Core/Rpc/RemoteBuildHost.cs

```diff
@@ -23,9 +23,6 @@ public RemoteBuildHost(RpcClient client)
     public Task<bool> HasUsableMSBuildAsync(string projectOrSolutionFilePath, CancellationToken cancellationToken)
         => _client.InvokeAsync<bool>(BuildHostTargetObject, nameof(IBuildHost.HasUsableMSBuild), parameters: [projectOrSolutionFilePath], cancellationToken);
 
-    public Task<ImmutableArray<(string ProjectPath, string ProjectGuid)>> GetProjectsInSolutionAsync(string solutionFilePath, CancellationToken cancellationToken)
-        => _client.InvokeAsync<ImmutableArray<(string ProjectPath, string ProjectGuid)>>(BuildHostTargetObject, nameof(IBuildHost.GetProjectsInSolution), parameters: [solutionFilePath], cancellationToken);
```

Comment:
It looks like this PR means that we will be getting the projects in a solution in-proc again, instead of thru an RPC call? Is that right?

Reply:
Yes, I should update the description. I ended up needing to pull in the SLNX work after realizing that we needed the Microsoft.Build dependency for solution parsing